app/views/header.js

```diff
@@ -27,6 +28,11 @@ const Header = Mn.ItemView.extend({
             'update reset',
             this.render
         );
+
+       this.listenTo(application.appState, 'change:currentPlaylist', () => {
+            let drawer = document.querySelector('[role="toolbar"]')
```

Comment:
Ideally, you shouldn't manipulate DOM nodes external to the current view. Prefer use bus events and bind actions on them in the concerned views. It'll so unbreak concerns.